src/gui/menu_manager.cpp

```diff
@@ -278,6 +278,19 @@ MenuManager::pop_menu()
   }
 }
 
+void
+MenuManager::pop_stack()
+{
+	if (m_menu_stack.empty())
+  {
+    log_warning << "trying to pop on an empty menu_stack" << std::endl;
+  }
+	else
+	{
+    m_menu_stack.pop_back();
+	}
+}
+
```

Comment:
Is this to avoid transitions when changing pages in the add-on list? If so, it would be better to add an argument to `pop_menu` and `push_menu` to always skip the transition, as having two similarly-named functions can cause confusion.